<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 3.0.2 (Python 3.6.2 on win32)" generated="20210202 16:09:35.516">
<suite id="s1" name="Connection Check" source="C:\Users\palak\Documents\coe-application-security\TestCases\connection_check.robot">
<test id="s1-t1" name="TC_001">
<kw name="Log To Console" library="BuiltIn">
<doc>Logs the given message to the console.</doc>
<arguments>
<arg>"Verifying Batch Profile API is functionally working"</arg>
</arguments>
<status status="PASS" starttime="20210202 16:09:37.076" endtime="20210202 16:09:37.077"></status>
</kw>
<kw name="Execute API" library="API">
<arguments>
<arg>${Excel_Location}</arg>
<arg>Agent</arg>
<arg>ACMS_Batch_Profiles_API</arg>
</arguments>
<kw name="Log To Console" library="BuiltIn">
<doc>Logs the given message to the console.</doc>
<arguments>
<arg>"Started API Execution"</arg>
</arguments>
<status status="PASS" starttime="20210202 16:09:37.078" endtime="20210202 16:09:37.078"></status>
</kw>
<kw name="API Wert" library="REST">
<arguments>
<arg>${API_Excel_Location}</arg>
<arg>${API_Excel_Sheet_Name}</arg>
<arg>${API_Module_Name}</arg>
</arguments>
<assign>
<var>${result}</var>
</assign>
<msg timestamp="20210202 16:09:37.127" level="INFO">Opening Excel
Opened Excel
Title of Sheet = Agent
Row 1 = API_Name
Row 2 = GATEWAY_API_POST
Row 3 = GATEWAY_API_GET
Row 4 = GATEWAY_API_DELETE
Row 5 = Mapping_API
Row 6 = Packages_API
Row 7 = Asset_API
Row 8 = ACMS_Summary_API
Row 9 = ACMS_Batch_Profiles_API
Module Found : ACMS_Batch_Profiles_API in Row - 9 of Worksheet - Agent
ACMS_Batch_Profiles_API
ACMS_Batch_Profiles_API
ACMS_Batch_Profiles_API
GET
GET
http
http

BaseURL = internal-qaplatformagentconfigurationserv-1810934581.us-east-1.elb.amazonaws.com

RelativeURL = /agent-configuration/v1/batchProfiles

http://internal-qaplatformagentconfigurationserv-1810934581.us-east-1.elb.amazonaws.com/agent-configuration/v1/batchProfiles
Body = {"createdTime":"2020-06-04T13:49:16.401Z","modifiedTime":"2020-06-04T13:49:16.401Z","id":"2e6f6ae4-a66a-11ea-898b-12e97a1f6711","description":"platform-hyperv-plugin","tag":"rmm-core","targets":["partnerID":"$1683$"}],"configurations":[{"packageName":"$platform-hyperv-plugin$"}],"createdBy":"santoshiMandloi","modifiedBy":"santoshiMandloi"}

Expecting ',' delimiter: line 1 column 207 (char 206)
Data from find_vulnerable_parameters 
{}
'Body'
'Header'
'Cookie'
http://internal-qaplatformagentconfigurationserv-1810934581.us-east-1.elb.amazonaws.com/agent-configuration/v1/batchProfiles
local variable 'Body' referenced before assignment
local variable 'Header' referenced before assignment
local variable 'Cookie' referenced before assignment
Found GET API, So Executing It.
local variable 'Body' referenced before assignment</msg>
<msg timestamp="20210202 16:09:37.128" level="INFO">File "c:\python36\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\python36\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "c:\python36\lib\site-packages\robot\run.py", line 550, in &lt;module&gt;
    run_cli(sys.argv[1:])
  File "c:\python36\lib\site-packages\robot\run.py", line 489, in run_cli
    return RobotFramework().execute_cli(arguments, exit=exit)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 46, in execute_cli
    rc = self._execute(arguments, options)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 83, in _execute
    rc = self.main(arguments, **options)
  File "c:\python36\lib\site-packages\robot\run.py", line 445, in main
    result = suite.run(settings)
  File "c:\python36\lib\site-packages\robot\running\model.py", line 248, in run
    self.visit(runner)
  File "c:\python36\lib\site-packages\robot\model\testsuite.py", line 161, in visit
    visitor.visit_suite(self)
  File "c:\python36\lib\site-packages\robot\model\visitor.py", line 87, in visit_suite
    suite.tests.visit(self)
  File "c:\python36\lib\site-packages\robot\model\itemlist.py", line 76, in visit
    item.visit(visitor)
  File "c:\python36\lib\site-packages\robot\model\testcase.py", line 74, in visit
    visitor.visit_test(self)
  File "c:\python36\lib\site-packages\robot\running\runner.py", line 135, in visit_test
    test.template).run_steps(test.keywords.normal)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 36, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 57, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 54, in run
    return_value = self._run(context, kw.args, result)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 80, in _run
    exception, return_ = self._execute(context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 144, in _execute
    StepRunner(context).run_steps(handler.keywords)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 36, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 57, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 54, in run
    return_value = self._run(context, kw.args)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 77, in _run
    return self._run_with_output_captured_and_signal_monitor(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 96, in _run_with_output_captured_and_signal_monitor
    return self._run_with_signal_monitoring(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 101, in _run_with_signal_monitoring
    return runner()
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 89, in &lt;lambda&gt;
    return lambda: handler(*positional, **named)
  File "C:\Users\palak\Documents\coe-application-security\MethodFiles\REST.py", line 142, in API_wert
    returnvalue = readexcel(Excel_Location, Excel_Sheet_Name, Module_Name)
  File "C:\Users\palak\Documents\coe-application-security\MethodFiles\REST.py", line 102, in readexcel
    traceback.print_stack()
  File "c:\python36\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\python36\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "c:\python36\lib\site-packages\robot\run.py", line 550, in &lt;module&gt;
    run_cli(sys.argv[1:])
  File "c:\python36\lib\site-packages\robot\run.py", line 489, in run_cli
    return RobotFramework().execute_cli(arguments, exit=exit)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 46, in execute_cli
    rc = self._execute(arguments, options)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 83, in _execute
    rc = self.main(arguments, **options)
  File "c:\python36\lib\site-packages\robot\run.py", line 445, in main
    result = suite.run(settings)
  File "c:\python36\lib\site-packages\robot\running\model.py", line 248, in run
    self.visit(runner)
  File "c:\python36\lib\site-packages\robot\model\testsuite.py", line 161, in visit
    visitor.visit_suite(self)
  File "c:\python36\lib\site-packages\robot\model\visitor.py", line 87, in visit_suite
    suite.tests.visit(self)
  File "c:\python36\lib\site-packages\robot\model\itemlist.py", line 76, in visit
    item.visit(visitor)
  File "c:\python36\lib\site-packages\robot\model\testcase.py", line 74, in visit
    visitor.visit_test(self)
  File "c:\python36\lib\site-packages\robot\running\runner.py", line 135, in visit_test
    test.template).run_steps(test.keywords.normal)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 36, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 57, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 54, in run
    return_value = self._run(context, kw.args, result)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 80, in _run
    exception, return_ = self._execute(context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 144, in _execute
    StepRunner(context).run_steps(handler.keywords)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 36, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 57, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 54, in run
    return_value = self._run(context, kw.args)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 77, in _run
    return self._run_with_output_captured_and_signal_monitor(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 96, in _run_with_output_captured_and_signal_monitor
    return self._run_with_signal_monitoring(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 101, in _run_with_signal_monitoring
    return runner()
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 89, in &lt;lambda&gt;
    return lambda: handler(*positional, **named)
  File "C:\Users\palak\Documents\coe-application-security\MethodFiles\REST.py", line 177, in API_wert
    traceback.print_stack()
  File "c:\python36\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\python36\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "c:\python36\lib\site-packages\robot\run.py", line 550, in &lt;module&gt;
    run_cli(sys.argv[1:])
  File "c:\python36\lib\site-packages\robot\run.py", line 489, in run_cli
    return RobotFramework().execute_cli(arguments, exit=exit)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 46, in execute_cli
    rc = self._execute(arguments, options)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 83, in _execute
    rc = self.main(arguments, **options)
  File "c:\python36\lib\site-packages\robot\run.py", line 445, in main
    result = suite.run(settings)
  File "c:\python36\lib\site-packages\robot\running\model.py", line 248, in run
    self.visit(runner)
  File "c:\python36\lib\site-packages\robot\model\testsuite.py", line 161, in visit
    visitor.visit_suite(self)
  File "c:\python36\lib\site-packages\robot\model\visitor.py", line 87, in visit_suite
    suite.tests.visit(self)
  File "c:\python36\lib\site-packages\robot\model\itemlist.py", line 76, in visit
    item.visit(visitor)
  File "c:\python36\lib\site-packages\robot\model\testcase.py", line 74, in visit
    visitor.visit_test(self)
  File "c:\python36\lib\site-packages\robot\running\runner.py", line 135, in visit_test
    test.template).run_steps(test.keywords.normal)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 36, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 57, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 54, in run
    return_value = self._run(context, kw.args, result)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 80, in _run
    exception, return_ = self._execute(context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 144, in _execute
    StepRunner(context).run_steps(handler.keywords)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 36, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 57, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 54, in run
    return_value = self._run(context, kw.args)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 77, in _run
    return self._run_with_output_captured_and_signal_monitor(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 96, in _run_with_output_captured_and_signal_monitor
    return self._run_with_signal_monitoring(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 101, in _run_with_signal_monitoring
    return runner()
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 89, in &lt;lambda&gt;
    return lambda: handler(*positional, **named)
  File "C:\Users\palak\Documents\coe-application-security\MethodFiles\REST.py", line 183, in API_wert
    traceback.print_stack()
  File "c:\python36\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\python36\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "c:\python36\lib\site-packages\robot\run.py", line 550, in &lt;module&gt;
    run_cli(sys.argv[1:])
  File "c:\python36\lib\site-packages\robot\run.py", line 489, in run_cli
    return RobotFramework().execute_cli(arguments, exit=exit)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 46, in execute_cli
    rc = self._execute(arguments, options)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 83, in _execute
    rc = self.main(arguments, **options)
  File "c:\python36\lib\site-packages\robot\run.py", line 445, in main
    result = suite.run(settings)
  File "c:\python36\lib\site-packages\robot\running\model.py", line 248, in run
    self.visit(runner)
  File "c:\python36\lib\site-packages\robot\model\testsuite.py", line 161, in visit
    visitor.visit_suite(self)
  File "c:\python36\lib\site-packages\robot\model\visitor.py", line 87, in visit_suite
    suite.tests.visit(self)
  File "c:\python36\lib\site-packages\robot\model\itemlist.py", line 76, in visit
    item.visit(visitor)
  File "c:\python36\lib\site-packages\robot\model\testcase.py", line 74, in visit
    visitor.visit_test(self)
  File "c:\python36\lib\site-packages\robot\running\runner.py", line 135, in visit_test
    test.template).run_steps(test.keywords.normal)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 36, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 57, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 54, in run
    return_value = self._run(context, kw.args, result)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 80, in _run
    exception, return_ = self._execute(context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 144, in _execute
    StepRunner(context).run_steps(handler.keywords)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 36, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 57, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 54, in run
    return_value = self._run(context, kw.args)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 77, in _run
    return self._run_with_output_captured_and_signal_monitor(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 96, in _run_with_output_captured_and_signal_monitor
    return self._run_with_signal_monitoring(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 101, in _run_with_signal_monitoring
    return runner()
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 89, in &lt;lambda&gt;
    return lambda: handler(*positional, **named)
  File "C:\Users\palak\Documents\coe-application-security\MethodFiles\REST.py", line 189, in API_wert
    traceback.print_stack()
  File "c:\python36\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\python36\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "c:\python36\lib\site-packages\robot\run.py", line 550, in &lt;module&gt;
    run_cli(sys.argv[1:])
  File "c:\python36\lib\site-packages\robot\run.py", line 489, in run_cli
    return RobotFramework().execute_cli(arguments, exit=exit)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 46, in execute_cli
    rc = self._execute(arguments, options)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 83, in _execute
    rc = self.main(arguments, **options)
  File "c:\python36\lib\site-packages\robot\run.py", line 445, in main
    result = suite.run(settings)
  File "c:\python36\lib\site-packages\robot\running\model.py", line 248, in run
    self.visit(runner)
  File "c:\python36\lib\site-packages\robot\model\testsuite.py", line 161, in visit
    visitor.visit_suite(self)
  File "c:\python36\lib\site-packages\robot\model\visitor.py", line 87, in visit_suite
    suite.tests.visit(self)
  File "c:\python36\lib\site-packages\robot\model\itemlist.py", line 76, in visit
    item.visit(visitor)
  File "c:\python36\lib\site-packages\robot\model\testcase.py", line 74, in visit
    visitor.visit_test(self)
  File "c:\python36\lib\site-packages\robot\running\runner.py", line 135, in visit_test
    test.template).run_steps(test.keywords.normal)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 36, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 57, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 54, in run
    return_value = self._run(context, kw.args, result)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 80, in _run
    exception, return_ = self._execute(context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 144, in _execute
    StepRunner(context).run_steps(handler.keywords)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 36, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 57, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 54, in run
    return_value = self._run(context, kw.args)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 77, in _run
    return self._run_with_output_captured_and_signal_monitor(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 96, in _run_with_output_captured_and_signal_monitor
    return self._run_with_signal_monitoring(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 101, in _run_with_signal_monitoring
    return runner()
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 89, in &lt;lambda&gt;
    return lambda: handler(*positional, **named)
  File "C:\Users\palak\Documents\coe-application-security\MethodFiles\REST.py", line 214, in API_wert
    traceback.print_stack()
  File "c:\python36\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\python36\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "c:\python36\lib\site-packages\robot\run.py", line 550, in &lt;module&gt;
    run_cli(sys.argv[1:])
  File "c:\python36\lib\site-packages\robot\run.py", line 489, in run_cli
    return RobotFramework().execute_cli(arguments, exit=exit)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 46, in execute_cli
    rc = self._execute(arguments, options)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 83, in _execute
    rc = self.main(arguments, **options)
  File "c:\python36\lib\site-packages\robot\run.py", line 445, in main
    result = suite.run(settings)
  File "c:\python36\lib\site-packages\robot\running\model.py", line 248, in run
    self.visit(runner)
  File "c:\python36\lib\site-packages\robot\model\testsuite.py", line 161, in visit
    visitor.visit_suite(self)
  File "c:\python36\lib\site-packages\robot\model\visitor.py", line 87, in visit_suite
    suite.tests.visit(self)
  File "c:\python36\lib\site-packages\robot\model\itemlist.py", line 76, in visit
    item.visit(visitor)
  File "c:\python36\lib\site-packages\robot\model\testcase.py", line 74, in visit
    visitor.visit_test(self)
  File "c:\python36\lib\site-packages\robot\running\runner.py", line 135, in visit_test
    test.template).run_steps(test.keywords.normal)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 36, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 57, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 54, in run
    return_value = self._run(context, kw.args, result)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 80, in _run
    exception, return_ = self._execute(context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 144, in _execute
    StepRunner(context).run_steps(handler.keywords)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 36, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 57, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 54, in run
    return_value = self._run(context, kw.args)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 77, in _run
    return self._run_with_output_captured_and_signal_monitor(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 96, in _run_with_output_captured_and_signal_monitor
    return self._run_with_signal_monitoring(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 101, in _run_with_signal_monitoring
    return runner()
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 89, in &lt;lambda&gt;
    return lambda: handler(*positional, **named)
  File "C:\Users\palak\Documents\coe-application-security\MethodFiles\REST.py", line 228, in API_wert
    traceback.print_stack()
  File "c:\python36\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\python36\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "c:\python36\lib\site-packages\robot\run.py", line 550, in &lt;module&gt;
    run_cli(sys.argv[1:])
  File "c:\python36\lib\site-packages\robot\run.py", line 489, in run_cli
    return RobotFramework().execute_cli(arguments, exit=exit)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 46, in execute_cli
    rc = self._execute(arguments, options)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 83, in _execute
    rc = self.main(arguments, **options)
  File "c:\python36\lib\site-packages\robot\run.py", line 445, in main
    result = suite.run(settings)
  File "c:\python36\lib\site-packages\robot\running\model.py", line 248, in run
    self.visit(runner)
  File "c:\python36\lib\site-packages\robot\model\testsuite.py", line 161, in visit
    visitor.visit_suite(self)
  File "c:\python36\lib\site-packages\robot\model\visitor.py", line 87, in visit_suite
    suite.tests.visit(self)
  File "c:\python36\lib\site-packages\robot\model\itemlist.py", line 76, in visit
    item.visit(visitor)
  File "c:\python36\lib\site-packages\robot\model\testcase.py", line 74, in visit
    visitor.visit_test(self)
  File "c:\python36\lib\site-packages\robot\running\runner.py", line 135, in visit_test
    test.template).run_steps(test.keywords.normal)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 36, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 57, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 54, in run
    return_value = self._run(context, kw.args, result)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 80, in _run
    exception, return_ = self._execute(context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 144, in _execute
    StepRunner(context).run_steps(handler.keywords)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 36, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 57, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 54, in run
    return_value = self._run(context, kw.args)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 77, in _run
    return self._run_with_output_captured_and_signal_monitor(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 96, in _run_with_output_captured_and_signal_monitor
    return self._run_with_signal_monitoring(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 101, in _run_with_signal_monitoring
    return runner()
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 89, in &lt;lambda&gt;
    return lambda: handler(*positional, **named)
  File "C:\Users\palak\Documents\coe-application-security\MethodFiles\REST.py", line 241, in API_wert
    traceback.print_stack()
  File "c:\python36\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\python36\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "c:\python36\lib\site-packages\robot\run.py", line 550, in &lt;module&gt;
    run_cli(sys.argv[1:])
  File "c:\python36\lib\site-packages\robot\run.py", line 489, in run_cli
    return RobotFramework().execute_cli(arguments, exit=exit)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 46, in execute_cli
    rc = self._execute(arguments, options)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 83, in _execute
    rc = self.main(arguments, **options)
  File "c:\python36\lib\site-packages\robot\run.py", line 445, in main
    result = suite.run(settings)
  File "c:\python36\lib\site-packages\robot\running\model.py", line 248, in run
    self.visit(runner)
  File "c:\python36\lib\site-packages\robot\model\testsuite.py", line 161, in visit
    visitor.visit_suite(self)
  File "c:\python36\lib\site-packages\robot\model\visitor.py", line 87, in visit_suite
    suite.tests.visit(self)
  File "c:\python36\lib\site-packages\robot\model\itemlist.py", line 76, in visit
    item.visit(visitor)
  File "c:\python36\lib\site-packages\robot\model\testcase.py", line 74, in visit
    visitor.visit_test(self)
  File "c:\python36\lib\site-packages\robot\running\runner.py", line 135, in visit_test
    test.template).run_steps(test.keywords.normal)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 36, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 57, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 54, in run
    return_value = self._run(context, kw.args, result)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 80, in _run
    exception, return_ = self._execute(context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 144, in _execute
    StepRunner(context).run_steps(handler.keywords)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 36, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 57, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 54, in run
    return_value = self._run(context, kw.args)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 77, in _run
    return self._run_with_output_captured_and_signal_monitor(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 96, in _run_with_output_captured_and_signal_monitor
    return self._run_with_signal_monitoring(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 101, in _run_with_signal_monitoring
    return runner()
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 89, in &lt;lambda&gt;
    return lambda: handler(*positional, **named)
  File "C:\Users\palak\Documents\coe-application-security\MethodFiles\REST.py", line 256, in API_wert
    traceback.print_stack()</msg>
<msg timestamp="20210202 16:09:37.247" level="INFO">${result} = {}</msg>
<status status="PASS" starttime="20210202 16:09:37.078" endtime="20210202 16:09:37.247"></status>
</kw>
<kw name="Log To Console" library="BuiltIn">
<doc>Logs the given message to the console.</doc>
<arguments>
<arg>"Executed Sucessfully"</arg>
</arguments>
<status status="PASS" starttime="20210202 16:09:37.248" endtime="20210202 16:09:37.248"></status>
</kw>
<kw name="List Should Not Contain Value" library="Collections">
<doc>Fails if the ``value`` is found from ``list``.</doc>
<arguments>
<arg>${result}</arg>
<arg>200</arg>
</arguments>
<status status="PASS" starttime="20210202 16:09:37.248" endtime="20210202 16:09:37.248"></status>
</kw>
<status status="PASS" starttime="20210202 16:09:37.077" endtime="20210202 16:09:37.249"></status>
</kw>
<status status="PASS" starttime="20210202 16:09:37.076" endtime="20210202 16:09:37.249" critical="yes"></status>
</test>
<status status="PASS" starttime="20210202 16:09:35.518" endtime="20210202 16:09:37.258"></status>
</suite>
<statistics>
<total>
<stat pass="1" fail="0">Critical Tests</stat>
<stat pass="1" fail="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="1" fail="0" id="s1" name="Connection Check">Connection Check</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20210202 16:09:35.551" level="ERROR">Error in file 'C:\Users\palak\Documents\coe-application-security\TestCases\connection_check.robot': Importing test library 'SeleniumLibrary' failed: ImportError: cannot import name 'PY_VERSION'
Traceback (most recent call last):
  File "c:\python36\lib\site-packages\SeleniumLibrary\__init__.py", line 26, in &lt;module&gt;
    from robotlibcore import DynamicCore
  File "c:\python36\lib\site-packages\robotlibcore.py", line 26, in &lt;module&gt;
    from robot.utils import PY_VERSION
PYTHONPATH:
  c:\python36\python36.zip
  c:\python36\DLLs
  c:\python36\lib
  c:\python36
  c:\python36\lib\site-packages</msg>
<msg timestamp="20210202 16:09:35.581" level="ERROR">Error in file 'C:\Users\palak\Documents\coe-application-security\Resources\API.robot': Importing test library 'SeleniumLibrary' failed: ImportError: cannot import name 'PY_VERSION'
Traceback (most recent call last):
  File "c:\python36\lib\site-packages\SeleniumLibrary\__init__.py", line 26, in &lt;module&gt;
    from robotlibcore import DynamicCore
  File "c:\python36\lib\site-packages\robotlibcore.py", line 26, in &lt;module&gt;
    from robot.utils import PY_VERSION
PYTHONPATH:
  c:\python36\python36.zip
  c:\python36\DLLs
  c:\python36\lib
  c:\python36
  c:\python36\lib\site-packages</msg>
<msg timestamp="20210202 16:09:35.590" level="ERROR">Error in file 'C:\Users\palak\Documents\coe-application-security\Resources\API.robot': Importing test library 'JSpringBotGlobal' failed: ModuleNotFoundError: No module named 'JSpringBotGlobal'
Traceback (most recent call last):
  None
PYTHONPATH:
  c:\python36\python36.zip
  c:\python36\DLLs
  c:\python36\lib
  c:\python36
  c:\python36\lib\site-packages</msg>
</errors>
</robot>
